Objects/typeobject.c

```diff
@@ -4344,7 +4344,8 @@ _PyObject_GetState(PyObject *obj, int required)
 
         if (required && Py_TYPE(obj)->tp_itemsize) {
             PyErr_Format(PyExc_TypeError,
-                         "cannot pickle '%.200s' object",
+                         "cannot pickle '%.200s' object: expected " \
+                         "fixed-length instance (tp_itemsize != 0)",
```

Comment:
An alternative message here could be "missing __getstate__ on variable-length object".

Reply:
Done, thank you.